packages/components/table/src/table-body/styles-helper.ts

```diff
@@ -98,7 +98,7 @@ function useStyles<T>(props: Partial<TableBodyProps<T>>) {
       )
     }
     classes.push(ns.e('cell'))
-    return classes.join(' ')
+    return classes.filter((className) => !!className).join(' ')
```

Comment:
**Suggestion:** 

use `Boolean(val)` instead of `!!val` for converting here.

Reply:
done